package structural.facadePattern.device;

public class MusicPlayer {

    public void initializeAudioDrivers() {
        System.out.println("Audio drivers initialized.");
    }

    public void decodeAudio() {
        System.out.println("Audio decoded.");
    }

    public void startPlayback() {
        System.out.println("Music playback started.");
    }
}
